www/src/components/unbird.js

```diff
@@ -0,0 +1,239 @@
+  border-radius: 3px;
+  overflow: hidden;
+  flex-direction: column;
+  justify-content: space-between;
+  text-align: center;
+  font-family: ${options.systemFontFamily.join(`,`)};
+`
+
+const Label = styled(`label`)`
+  width: 100%;
+  height: 240px;
+  background-color: ${colors.gatsby};
+  color: #fff;
+  font-weight: 200;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 40px;
+  font-size: 22px;
+`
+
+const Form = styled(`form`)`
+  height: 100%;
+  position: relative;
+`
+
+const Input = styled(`input`)`
+  height: 60px;
+  width: 100%;
+  font-size: 14px;
+  padding: 20px 60px 20px 20px;
+  border: none;
+  resize: none;
+
+  &:hover {
+    outline: none;
+  }
+`
+
+const Send = styled(`button`)`
+  width: 30px;
+  height: 30px;
+  position: absolute;
+  right: 20px;
+  bottom: 15px;
+  cursor: pointer;
+  background-size: 30px 30px;
+  background-repeat: no-repeat;
+  background-position: center;
+  border: none;
+`
+
+class Unbird extends React.Component {
+  state = {
+    visible: false,
+    feedbackInput: ``,
+    statusMessage: ``,
+  }
+
+  static defaultProps = {
+    feedbackPrompt: `How can we improve your experience?`,
+    feedbackPlaceholder: `Send feedback...`,
+  }
+
+  render() {
+    return (
+      <FeedbackComponent>
+        <FeedbackToggle onClick={this.toggleFeedbackForm}>
+          <IconWrapper>
+            {this.state.visible ? (
+              <CancelIcon
+                css={{
+                  color: `#fff`,
+                  fontSize: rhythm(1.6),
+                  padding: rhythm(0.2),
+                  position: `absolute`,
+                  top: `50%`,
+                  left: `50%`,
+                  transform: `translate(-50%, -50%)`,
+                  cursor: `pointer`,
+                }}
+              />
+            ) : (
+              <EnvelopeIcon
+                css={{
+                  color: `#fff`,
+                  fontSize: rhythm(1.4),
+                  padding: rhythm(0.2),
+                  position: `absolute`,
+                  top: `50%`,
+                  left: `50%`,
+                  transform: `translate(-50%, -50%)`,
+                  cursor: `pointer`,
+                }}
+              />
+            )}
+          </IconWrapper>
+        </FeedbackToggle>
+
+        {this.state.visible && (
+          <FeedbackForm>
+            <Form autoComplete="off" onSubmit={this.submitFeedback}>
+              <Label htmlFor="unbird-feedback">
+                {this.props.feedbackPrompt}
+              </Label>
+              {this.state.statusMessage && (
+                <StatusMessage>{this.state.statusMessage}</StatusMessage>
+              )}
+              <Input
+                id="unbird-feedback"
+                type="text"
+                value={this.state.feedbackInput}
+                onChange={this.handleFeedbackInput}
+                placeholder={this.props.feedbackPlaceholder}
```

Comment:
Maybe add a `required` attribute so we can get validation "for free?" Right now this would be able to submit an empty form, right?

Reply:
e.g. see [this CodeSandbox](https://codesandbox.io/s/21445xmn8j)